Instastories.py

```diff
@@ -319,15 +319,16 @@ def start_scrape(scrape_settings, user_limit, media_mode="all", ids_source="all"
         stories_ids = tray_to_ids(stories)
         if user_limit <= 0: user_limit = len(stories_ids)
     extra_ids = normalize_extra_ids(scrape_settings["extra_ids"] if "extra_ids" in scrape_settings else [])
-    ids = get_ids(stories_ids, user_limit, extra_ids, ids_source)
+    blacklisted_ids = normalize_extra_ids(scrape_settings['blacklisted_ids'])
```

Comment:
Maybe rename the function to something both applicable to extra id and blaclisted id? 

like: `normalize_ids`

Reply:
Done